### PR TITLE
chore(modal): add testID to modal

### PR DESCRIPTION
**Pull request by shamshad17**
- add testID to modal

### CODE REVIEW

File: packages/blade-old/src/molecules/Modal/Modal.native.js [hunk 1]
```diff
@@ -118,6 +118,7 @@ const Modal = ({ children, variant, visible, onClose, onBackdropClick }) => {
       <Space margin={[0]}>
         <Flex justifyContent="flex-end">
           <RNModal
+            testID="bottomsheet-modal"
```
Comment (gupta-ji6) [inline packages/blade-old/src/molecules/Modal/Modal.native.js:121]:
```suggestion
            testID="ds-bottomsheet-modal"
```
I think we add `ds` as a prefix as a convention to DS components, check once.

Also, how about adding `testID` as a prop & give a default value to it?

Comment (chaitanyadeorukhkar) [inline packages/blade-old/src/molecules/Modal/Modal.native.js:121]: Why not accept `testID` from props and do this? We're doing this for other components as well. Refer Button or Card
```suggestion
             {...automation(testID)}
```

**Review by gupta-ji6 — DISMISSED** [review]

File: packages/blade-old/src/molecules/Modal/Modal.native.js (hunk 1 above)
Comment (gupta-ji6) [reply packages/blade-old/src/molecules/Modal/Modal.native.js]: looks good now, resolving

**Review by chaitanyadeorukhkar — APPROVED** [review]

File: .changeset/breezy-icons-obey.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@razorpay/blade": patch
```
Comment (chaitanyadeorukhkar) [inline .changeset/breezy-icons-obey.md:2]: Why are you bumping blade?
```suggestion
```

**Review by chaitanyadeorukhkar — CHANGES_REQUESTED** [review]

File: .changeset/breezy-icons-obey.md (hunk 2 above)
Comment (shamshad17) [reply .changeset/breezy-icons-obey.md]: deleted

**Review by chaitanyadeorukhkar — APPROVED** [review]
Added a changeset to only bump blade-old

**Pull request merged**